go.mod

```diff
@@ -36,7 +37,7 @@ replace (
 	github.com/jteeuwen/go-bindata => github.com/jteeuwen/go-bindata v3.0.8-0.20151023091102-a0ff2567cfb7+incompatible
 	github.com/openshift/api => github.com/openshift/api v0.0.0-20191101174058-d428681bd3ce
 	github.com/openshift/client-go => github.com/openshift/client-go v0.0.0-20191022152013-2823239d2298
-	github.com/openshift/library-go => github.com/openshift/library-go v0.0.0-20191121124438-7c776f7cc17a
+	github.com/openshift/library-go => github.com/sallyom/library-go v0.0.0-20191125154451-7578d03d01df
```

Comment:
only for testing?

Reply:
yes, only for testing, will remove once library-go pr is merged.. i'll hold until then.